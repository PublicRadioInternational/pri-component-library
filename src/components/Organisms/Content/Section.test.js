/**
 * @file Section.test.js
 * Contains tests for Section.component.js.
 */

import React from 'react';
import renderer from 'react-test-renderer';

import Section from './Section.component';

describe('<Section />', () => {
  it('Matches the Section snapshot', () => {
    const component = renderer.create(<Section />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
