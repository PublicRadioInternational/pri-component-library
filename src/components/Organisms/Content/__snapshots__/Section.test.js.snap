// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Section /> Matches the Section snapshot 1`] = `
<section
  className="className && null"
/>
`;
